ros_buildfarm/templates/release/rpm/mock_config.cfg.em

```diff
@@ -0,0 +1,50 @@
+include('/etc/mock/default.cfg')
+
+# Change the root name since we're modifying the chroot
+config_opts['root'] += '-ros-buildfarm'
+
+# Add python3-rpm-macros to resolve %{python3_pkgversion} in rosdep rules
+config_opts['chroot_setup_cmd'] += ' python3-rpm-macros'
+
+# Install weak dependencies to get group members
+config_opts['yum_builddep_opts'] = config_opts.get('yum_builddep_opts', []) + ['--setopt=install_weak_deps=True']
+config_opts['dnf_builddep_opts'] = config_opts.get('dnf_builddep_opts', []) + ['--setopt=install_weak_deps=True']
+config_opts['microdnf_builddep_opts'] = config_opts.get('microdnf_builddep_opts', []) + ['--setopt=install_weak_deps=True']
+
+# Disable debug packages until infrastructure can handle it
+config_opts['macros']['debug_package'] = '%{nil}'
+
+# Hack the %{dist} macro to allow release suffixing
+config_opts['macros']['%dist'] = '.' + config_opts['dist'] + '%{?dist_suffix}'
```

Comment:
This isn't ideal. `config_opts['dist']` doesn't necessarily correspond to the original value of `%{dict}`, but it seems to be the same for Fedora and CentOS builds.

There isn't a surefire way to "append" to an RPM macro with "define" semantics (as opposed to "global" semantics, which can be used inside of a spec file).

The better way to address this is ros-infrastructure/bloom#569. Once that change propagates, we can switch to that.